Tabloid/Controllers/PostController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Tabloid.Repositories;
+using System.Collections.Generic;
+using Tabloid.Models;
+using System;
+
+namespace Tabloid.Controllers
+{
```

Comment:
May want to add [Authorize] here.

Reply:
Added [Authorize] to the PostController per review notes.